lib/msf/core/payload/windows/reverse_http.rb

```diff
@@ -39,7 +39,8 @@ def generate(opts={})
       host:        ds['LHOST'] || '127.127.127.127',
       port:        ds['LPORT'],
       retry_count: ds['StagerRetryCount'],
-      retry_wait:  ds['StagerRetryWait']
+      retry_wait:  ds['StagerRetryWait'],
+      size:        ds['StagerStagePayloadSize'] || '0x400000'
```

Comment:
One more as Integer value:
```suggestion
      size:        ds['StagerStagePayloadSize'] || 0x400000
```